Makefile

```diff
@@ -2,7 +2,7 @@
 TEST_INFRA_VERSION ?= latest
 
 # Version of gRPC core used for the gRPC driver
-DRIVER_VERSION ?= v1.58.1
+DRIVER_VERSION ?= v1.60.0
```

Comment:
Could you change this version to `v1.62.1`? Thanks!

Reply:
@fdfzcq @paulosjca  FYI I made the same change in #390 that this PR is based on. I suppose the easiest is to get #390 merged first, then rebase this one. In any case it's an easy conflict to resolve.

Reply:
I removed the commits from @lepistone in this PR now